ingestion/src/metadata/workflow/base.py

```diff
@@ -106,6 +108,12 @@ def __init__(
 
         self.post_init()
 
+    @property
+    def output_handler(self) -> WorkflowOutputHandler:
+        if not self._output_handler:
+            self._output_handler = WorkflowOutputHandler()
+        return self._output_handler
```

Comment:
Could we instantiate it with the class in the `__init__` -- or even pass it as an argument to where we instantiate that class (so users could swap output handler if ever needed)?

Reply:
I would be fine passing it as an argument in the init and maybe having the logic there as well.